spec/features/merchants/items/index_spec.rb

```diff
@@ -72,4 +72,62 @@
       end
     end
   end
+
+  describe "US 10 - items grouped  by status" do
```

Comment:
I like the clear naming convention you have going here that specifies the user story. Makes sense for other users to review.